### PR TITLE
Convert to Markdown all DSL levels using HTML that can be trivially converted

### PR DESCRIPTION
**Pull request by Hamms**
Part of the process of removing HTML from DSL fields:

1. Enable support for markdown in relevant DSL fields (https://github.com/code-dot-org/code-dot-org/pull/32750)
2. Update all DSL levels to use markdown rather than HTML
   1. levels whose HTML can be directly converted to markdown (links, bold, italic, etc) **<- this PR**
   2. levels whose HTML is already close to markdown (manually-styled code blocks, etc)
   3. levels that will need to be manually refactored
3. Disable HTML

Note that in addition to converting HTML syntax to markdown, this change also normalizes a bunch of whitespace; these changes aren't strictly necessary, but they will make it easier to identify which levels are actually using HTML syntax.

## Links

- [jira](https://codedotorg.atlassian.net/browse/FND-904)

## Testing story

As with https://github.com/code-dot-org/code-dot-org/pull/32750, I rendered all affected levels to HTML both before and after this change, and compared the results. In this case, there were no changes whatsoever (which is to be expected, as this PR represents the "trivial" changes)

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]
I did a few spot checks and everything I saw looks good.

File: dashboard/config/scripts/cspu5_u3l13_debugging_multi.multi [hunk 1]
```diff
@@ -19,4 +19,4 @@ pJSN1M6T6FmomzeoU/es4nx8BCbtH6aLl4oqKWcDrIHc0Z6skFdqRpHOxiJp
 ltLgKVSnVGBQQp/Ts4rBhhcN2NQfBCoU7NavEYdLTHhaQjGSunC8dIsSgHoL
 tAAwpH40ENqjjO+tGeGcQOH+MkIzNg3zK+LTpCQRFpNVlbcnCQIjJYXlHX5v
 1ZL35nA4P98sNAqKd14QTCRp/xiIQ1k2ew==
-'
+'
```
Comment (dmcavoy) [inline dashboard/config/scripts/cspu5_u3l13_debugging_multi.multi:22]: Don't we want the new line at the end of the file?

File: dashboard/config/scripts/cspu5_u3l20_student_lesson_introduction.external [hunk 2]
```diff
@@ -37,4 +37,4 @@ KWnMU1s8ErF/AP+jax27UEhd95YabgpwZrnSIrmNIQNoe0uc/47YfxBD18Dk
 3Zhn8knbkljrPDAdyrnjs3WaFQpfgfjub8WdkuA5l0dlhwnItIdEpGlybJLp
 7vHmJQug95jFDcTDWtgWCy4DPFIsT5e46t0cEUn1+c2rhWOB41M2k5Lzlvp0
 7KZibtYNqD73MJBQlY5t+gp48r0qvXs4KF7aOn+D6Q==
-'
+'
```
Comment (dmcavoy) [inline dashboard/config/scripts/cspu5_u3l20_student_lesson_introduction.external:40]: Same question

File: dashboard/config/scripts/cspu5_u3l25_student_lesson_introduction.external [hunk 3]
```diff
@@ -32,4 +32,4 @@ M1Vm98Gqhqob39i2sGwusmMmRsehI+w+VxRGhgn+zlak9HsfT0Uy8jDsHfP6
 XvfZVSVX9fF5K12K/33iSa+Zjf+oK+3fUvPlXVJPiQdKW+3LUwygKuvbSEQX
 LpdBzlVM2vdKvNcz62EiyytDoz48ZrP7ta5yskVSlefZ6goYFMSRf2VsuXCT
 4PJRWFgb95nNXZLCuy5ZiQ==
-'
+'
```
Comment (dmcavoy) [inline dashboard/config/scripts/cspu5_u3l25_student_lesson_introduction.external:35]: same question

File: dashboard/config/scripts/cspu5_u3l27_student_lesson_introduction.external [hunk 4]
```diff
@@ -35,4 +35,4 @@ i68YWfx+RsF1vfBZ/aMLOuPpfQaX1SSY4FduVKCDa6RSwof5YMa94UXubA+M
 pWb99ao3MS1URYi0nYtdDNYVDNSEPYkHXLY1P+hcmqIMaUvrbuaVHjtGnwXW
 0UkXvJYbYclnihgHBfZ3afOAOLU3cxuIzTZzxJy7YprufeH3v30Vgy4rj9nA
 84IViRS9vG1WqFXT/MOmTg/3NKreXCmrVkZ1TFLMAj7NoFv8wEaN5ss=
-'
+'
```
Comment (dmcavoy) [inline dashboard/config/scripts/cspu5_u3l27_student_lesson_introduction.external:38]: same question

File: dashboard/config/scripts/cspu5_u3l28_student_lesson_introduction.external [hunk 5]
```diff
@@ -28,4 +28,4 @@ r+Cp5Ino0FLL5f898P4mJuFbr00ssxIevqfdb0s8PA8bdhC/OUN9ls9VCCeQ
 9rWj36kxWCDbIScLiH9nrnZlpfzf4JEqpeIxJsazlHLAz1I14KBp1piHRg30
 pHZEVFyVwsdXU0qz23jhWpfJMcuoSCTacxJlA4mnpWnONjNIeD3s3Zp1l2aR
 0AkXVcvl3MHkaS21PXH4yq0iLS8=
-'
+'
```
Comment (dmcavoy) [inline dashboard/config/scripts/cspu5_u3l28_student_lesson_introduction.external:31]: same question

**Review by dmcavoy — APPROVED** [review]
There where a couple files were you were getting rid of the newline at the end of the file. Was that on purpose?

**Comment by Hamms** [comment]
I don't think this change actually removed the newline at the end of any files, just some files are currently without a newline and github points that out.

Either way, it doesn't make a difference to the actual level

File: dashboard/config/scripts/cspu5_u3l13_debugging_multi.multi (hunk 1 above)
Comment (Hamms) [reply dashboard/config/scripts/cspu5_u3l13_debugging_multi.multi:22]: nope! Doesn't make a difference either way

**Pull request merged**